.github/workflows/dotnetcore.yml

```diff
@@ -30,9 +30,6 @@ jobs:
       with:
         dotnet-version: 8.0.x
 
-    - name: update versions
-      run: .\updateAssemblyInfo.ps1
-
     # Build and test
```

Comment:
the dotnet restore is not needed. It's done by dotnet test

Reply:
also, we should repalce the run .\pack.ps1 by dotnet pack --no-restore --no-build